ignite/engines/__init__.py

```diff
@@ -49,9 +49,9 @@ def create_supervised_evaluator(model, metrics={}, cuda=False):
     """
     def _inference(engine, batch):
         model.eval()
```

Comment:
This code could be a little different using no_grad function and the fact that Variable=Tensor
```
with torch.no_grad():
    x, y = _prepare_batch(batch, device)
    y_pred = model(x)
    return y_pred
```

Reply:
I think we should keep anyway `model.eval()`

Reply:
Yes, otherwise it will break some ops (e.g., BatchNorm, and Dropout) in inference mode.